airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/DebeziumRecordPublisher.java

```diff
@@ -156,6 +156,13 @@ protected static Properties getDebeziumProperties(JsonNode config, ConfiguredAir
     props.setProperty("slot.name", config.get("replication_method").get("replication_slot").asText());
     props.setProperty("publication.name", config.get("replication_method").get("publication").asText());
 
+    // By default "decimal.handing.mode=precise" which's caused returning this value as a binary.
+    // The "double" type may cause a loss of precision, so set Debezium's config to store it as a String
+    // explicitly in its Kafka messages for more details see:
+    // https://debezium.io/documentation/reference/connectors/mysql.html#mysql-decimal-types
```

Comment:
this line links to MySQL, would be good to have it pointing here:
https://debezium.io/documentation/reference/1.4/connectors/postgresql.html#postgresql-decimal-types